the-faseb-journal.csl

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0"
+    default-locale="en-US"
+    class="in-text" 
+    page-range-format="expanded"
+    demote-non-dropping-particle="sort-only"
+    initialize-with=". "
```

Comment:
please delete lines 7 and 8, then put the style element into a single line. Don't separate the attributes into new lines.